modules/api/php/endpoints/candidate/visit/instrument/flags.class.inc

```diff
@@ -95,9 +95,9 @@ class Flags extends Endpoint implements \LORIS\Middleware\ETagCalculator
             return $this->_handleGET($request);
 
         case 'PUT':
+            return $this->_handlePUT($request);
```

Comment:
```suggestion
            return $this->_handlePUT($request);

```

Reply:
It doesn't show as updated, but as you can see the empty line was added.